test/parallel/test-fs-watch-encoding.js

```diff
@@ -19,6 +19,9 @@ const common = require('../common');
 if (common.isAIX)
   common.skip('folder watch capability is limited in AIX.');
 
+if (common.isIBMi)
+  common.skip('IBMi does not support `fs.watch()`');
```

Comment:
how is it that this test passed prior to this PR? are we breaking an existing feature?

Reply:
Well, I don't know. The CI job for IBMi is not included in the `request-ci` flow. I added it to align with pre-existing watch cases.

Reply:
Hello! Not necessary to add this, as `common.isAIX` is also true on IBMi (Which would skip the test in the CI prior to this PR)

Reply:
Thanks @V-for-Vasili. Is there a reference to this in the Node codebase?

Reply:
It looks like `isAIX` is `process.platform === 'aix'`, and there is a comment in `test/common/index.js` that says:

```
// On IBMi, process.platform and os.platform() both return 'aix',
// It is not enough to differentiate between IBMi and real AIX system.
```